lib/dependency-tree/index.ts

```diff
@@ -25,6 +25,10 @@ export function buildTree(
     imageVersion = targetImage.slice(versionSeparator + 1);
   }
 
+  if (imageName.endsWith(".tar")) {
```

Comment:
shouldn't we check if it begins with `docker-archive` like in the second place we do it?

Reply:
We strip that from the `targetImage` so there is no prefix here